### PR TITLE
docs: v2 upgrade height for mainnet beta

### PR DESCRIPTION
**Pull request by rootulp**
I estimated this height using the blockheight tool from celestia-app repo. On mainnet the block time is **11.75 seconds**

```
$ go run main.go https://celestia-rpc.publicnode.com:443 2024-09-18T14:00:00
chainID: celestia
currentHeight: 2268730
currentTime: 2024-09-04 13:49:30.057435899 +0000 UTC
targetHeight: 2371728
targetTime: 2024-09-18 14:00:00 +0000 UTC
diffInSeconds: 1.210229e+06
diffInBlockHeight: 102998
```

Then we revised the estimate to a block height that is 5 blocks before a multiple of 1500 so that a state sync snapshot is created shortly after the upgrade.

```
$ go run main.go https://celestia-rpc.publicnode.com:443 2371495
chainID: celestia
currentHeight: 2269085
currentTime: 2024-09-04 14:58:54.478635553 +0000 UTC
diffInBlockHeight: 102410
diffInTime: 334h15m17.5s
arrivalTime: 2024-09-18 13:14:11.978635553 +0000 UTC
```

### CODE REVIEW

File: nodes/consensus-node.md [hunk 1]
```diff
@@ -303,7 +303,7 @@ If you are running celestia-app >= v2.0.0: then you'll want to start the node wi
 ::: code-group
 
 ```sh-vue [Mainnet Beta]
-celestia-appd start --v2-upgrade-height <height>
+celestia-appd start --v2-upgrade-height 2371728
```
Comment (jcstein) [inline nodes/consensus-node.md:306]: lgtm `2371728`

File: nodes/hardfork-process.md [hunk 2]
```diff
@@ -43,8 +43,8 @@ The Lemongrass hardfork is the first consensus layer breaking change since Celes
 - If you are a consensus node or validator operator: you will need to download and run a celestia-app binary >= v2.0.0 prior to the `--v2-upgrade-height` to remain on the canonical chain. You do not need to use a tool like [cosmovisor](https://docs.cosmos.network/main/build/tooling/cosmovisor) to upgrade the binary at the upgrade height.
 - If you are a DA node operator, you will need to download and run a compatible celestia-node binary >= v0.16.0-rc0 prior to the upgrade height.
 
-Network      | Chain ID   | Datetime                                 | `--v2-upgrade-height`
+Network      | Chain ID   | Date and approximate time                | `--v2-upgrade-height`
 -------------|------------|------------------------------------------|----------------------
 Arabica      | arabica-11 | 2024/08/19 @ 14:00 UTC                   | 1751707
 Mocha        | mocha-4    | 2024/08/28 @ 14:00 UTC                   | 2585031
-Mainnet Beta | celestia   | TBD approximately 2024/09/18 @ 14:00 UTC | TBD
+Mainnet Beta | celestia   | 2024/09/18 @ 14:00 UTC                   | 2371728
```
Comment (jcstein) [inline nodes/hardfork-process.md:50]: lgtm `2024/09/18 @ 14:00 UTC`

File: nodes/validator-node.md [hunk 3]
```diff
@@ -153,7 +153,7 @@ If you are running celestia-app >= v2.0.0: then you'll want to start the node wi
 ::: code-group
 
 ```sh-vue [Mainnet Beta]
-celestia-appd start --v2-upgrade-height <height>
+celestia-appd start --v2-upgrade-height 2371728
```
Comment (jcstein) [inline nodes/validator-node.md:156]: lgtm `2371728`

**Review by jcstein — APPROVED** [review]

**Comment by rootulp** [comment]
Based on a [Slack thread](https://celestia-team.slack.com/archives/C06RZVA9KH9/p1725458063308459) we're going to adjust the block height to something shortly before a state sync snapshot (a block height that is a multiple of 1500).

**Comment by rootulp** [comment]
On second thought, we don't need to merge this until DevOps is ready to upgrade Mainnet nodes so will convert to draft until then.

**Review by evan-forbes — APPROVED** [review]

**Review by jcstein — APPROVED** [review]
2371495

**Comment by rootulp** [comment]
Most recent block ETA based on 11.74 second block interval

```go
$ go run main.go https://celestia-rpc.publicnode.com:443 2371495
chainID: celestia
currentHeight: 2307473
currentTime: 2024-09-09 20:09:15.962493044 +0000 UTC
diffInBlockHeight: 64022
diffInTime: 208h46m58s
arrivalTime: 2024-09-18 12:56:13.962493044 +0000 UTC
```

**Pull request merged**